### PR TITLE
Add SVTR Tiny model

### PR DESCRIPTION
**Pull request by zhtmike**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [x] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [ ] You have added unit tests

## Motivation

(Write your motivation for proposed changes here.)

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

**Comment by zhtmike** [comment]
部分data transform目前跟本身有的会有重叠，会在 #300 重构。

File: configs/rec/svtr/README_CN.md
Comment (SamitHuang) [inline configs/rec/svtr/README_CN.md:1]: The result table should align center.

Comment (zhtmike) [reply configs/rec/svtr/README_CN.md:1]: Done.

**Review by SamitHuang — APPROVED** [review]

File: mindocr/models/backbones/rec_svtr.py [hunk 1]
```diff
@@ -0,0 +1,629 @@
...
+            x = self.dropout(x)
+
+        if self.use_lenhead:
+            return [x], [len_x]
+        else:
+            return [x]
+
+
+@register_backbone
+def rec_svtr(pretrained: bool = True, **kwargs):
```
Comment (HaoyangLee) [inline mindocr/models/backbones/rec_svtr.py:620]: Is the `pretrained` supposed to be `False` by default?

File: configs/rec/svtr/README_CN.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+```
+
+**模型评估的数据配置**
+
+我们使用 `evaluation/` 下的数据集作为基准数据集。在**每个单独的数据集**（例如 CUTE80、IC03_860 等）上，我们通过将数据集的目录设置为评估数据集来执行完整评估。这样，我们就得到了每个数据集对应精度的列表，然后报告的精度是这些值的平均值。
+
+如要重现报告的评估结果，您可以：
+- 方法 1：对所有单个数据集重复评估步骤：CUTE80、IC03_860、IC03_867、IC13_857、IC131015、IC15_1811、IC15_2077、IIIT5k_3000、SVT、SVTP。然后取平均分。
+
+- 方法 2：将所有基准数据集文件夹放在同一目录下，例如`评估/`。并使用脚本`tools/benchmarking/multi_dataset_eval.py`。
```
Comment (HaoyangLee) [inline configs/rec/svtr/README_CN.md:167]:
```suggestion
- 方法 2：将所有基准数据集文件夹放在同一目录下，例如`evaluation/`。并使用脚本`tools/benchmarking/multi_dataset_eval.py`。
```

File: configs/rec/svtr/README_CN.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+
+在数据处理时，真实文本会根据提供的字符字典转换为标签 ID，字典中键是字符，值是 ID。默认情况下，字典 **"0123456789abcdefghijklmnopqrstuvwxyz"**，这代表着id=0 将对应字符'0'。在默认设置下，字典只考虑数字和小写英文字符，不包括空格。
+
+
+### 内置词典
+
+Mindocr内置了一部分字典，均放在了 `mindocr/utils/dict/` 位置，可选择合适的字典使用。
+
+- `en_dict.txt` 是一个包含94个字符的英文字典，其中有数字，常用符号以及大小写的英文字母。
+- `ch_dict.txt` 是一个包含6623个字符的中文字典，其中有常用的繁简体中文，数字，常用符号以及大小写的英文字母
```
Comment (HaoyangLee) [inline configs/rec/svtr/README_CN.md:312]:
```suggestion
- `ch_dict.txt` 是一个包含6623个字符的中文字典，其中有常用的繁简体中文，数字，常用符号以及大小写的英文字母。
```

File: configs/rec/crnn/README.md [hunk 4]
```diff
@@ -37,21 +37,25 @@ Table Format:
...
 | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: |
 | CRNN (ours)      | D910x8-MS1.8-G | VGG7 | 82.03%    | 2445 s/epoch          | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_vgg7.yaml)     | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c-3a19e349.mindir)   |
 | CRNN (ours)      | D910x8-MS1.8-G | ResNet34_vd | 84.45%    | 2118 s/epoch         | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_resnet34.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_resnet34-83f37f07-2f016384.mindir) |
 | CRNN (PaddleOCR) | -              | ResNet34_vd | 83.99%           | -                      | -                                                                                              | -                                                                                          |
```
Comment (HaoyangLee) [inline configs/rec/crnn/README.md:46]: crnn rare svtr的中英文文档都删去与paddleocr的对比。

File: configs/rec/crnn/README.md [hunk 5]
```diff
@@ -37,21 +37,25 @@ Table Format:
 
 According to our experiments, the evaluation results on public benchmark datasets (IC03, IC13, IC15, IIIT, SVT, SVTP, CUTE) is as follow:
 
+<div align="center">
+
 | **Model** | **Context**    | **Backbone** | **Avg Accuracy** | **Train T.** | **Recipe** | **Download** | 
 | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: | :-----: |
 | CRNN (ours)      | D910x8-MS1.8-G | VGG7 | 82.03%    | 2445 s/epoch          | [yaml](https://github.com/mindspore-lab/mindocr/blob/main/configs/rec/crnn/crnn_vgg7.yaml)     | [ckpt](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c.ckpt) \| [mindir](https://download.mindspore.cn/toolkits/mindocr/crnn/crnn_vgg7-ea7e996c-3a19e349.mindir)   |
```
Comment (HaoyangLee) [inline configs/rec/crnn/README.md:44]: 删去**ours**，crnn rare svtr中英文文档同理。

File: mindocr/models/backbones/rec_svtr.py (hunk 1 above)
Comment (zhtmike) [reply mindocr/models/backbones/rec_svtr.py:620]: 没有一个通用的 pretrained weight，所有一般是靠url提供。

File: configs/rec/crnn/README.md (hunk 4 above)
Comment (zhtmike) [reply configs/rec/crnn/README.md]: Done.

File: configs/rec/svtr/README_CN.md (hunk 2 above)
Comment (zhtmike) [reply configs/rec/svtr/README_CN.md]: Updated.

File: configs/rec/svtr/README_CN.md (hunk 3 above)
Comment (zhtmike) [reply configs/rec/svtr/README_CN.md]: Updated.

File: configs/rec/crnn/README.md (hunk 5 above)
Comment (zhtmike) [reply configs/rec/crnn/README.md]: Done.

**Review by HaoyangLee — APPROVED** [review]

**Pull request merged**